test/__init__.py

```diff
@@ -0,0 +1,10 @@
+def _init_fb_ctypes():
+    # Initiaization required only in facebook infrastructure to use soundfile
+    import libfb.py.ctypesmonkeypatch
+    libfb.py.ctypesmonkeypatch.install()
+
+
+try:
+    _init_fb_ctypes()
+except Exception:
```

Comment:
nit: this would catch any exception, even ones that we may want to catch when on the fb infrastructure. i'm ok with this here for now though since this is not a regression for the community.

Reply:
I'm wondering if we should make this internal only. Doesn't seem relevant to the OSS community

Reply:
> I'm wondering if we should make this internal only. Doesn't seem relevant to the OSS community

I felt the same and tried to exclude it, but 1. `__init__.py` file is required (even if it's empty) to make the imports relative, and 2. `__init__.py` is the most appropriate place to perform fbcode initialization. (otherwise we have to do it everywhere.)
So, my judgement was rather than maintaining two different `__init__.py` for OSS and internal, it's better to have the same things, because there will be a time someone working on fbcode forgets that we keep two different things and they get tangled.